Tools/MiniBrowser/playstation/MainWindow.cpp

```diff
@@ -106,7 +115,45 @@ MainWindow::MainWindow(const char* requestedURL)
     }});
 
     createNewWebView(nullptr);
-    activeWebView()->loadURL(requestedURL ? requestedURL : "https://webkit.org");
+
+    // Parse command line options
+    string requestedURL;
+
+    for (auto option : options) {
+        bool foundOption = false;
+        __debugbreak();
+        for (auto optionName : s_optionNames) {
+            int len = strlen(optionName);
+            if (!option.compare(0, len, optionName)) {
+                foundOption = true;
+                string errorMsg = executeOption(optionName, option);
+                if (!errorMsg.empty()) {
+                    fprintf(stderr, "%s\n", errorMsg.c_str());
+                    return;
+                }
+                break;
+            }
+        }
+
+        if (!foundOption)
+            requestedURL = option;
+    }
+
+    activeWebView()->loadURL("https://webkit.org");
```

Comment:
Do you want to use `requestedURL` here?

Reply:
resolved in https://github.com/WebKit/WebKit/pull/5154/commits/ec72e16864b4383d8497aac77fa097414823a309